pkg/antctl/raw/multicluster/create/clusterclaim.go

```diff
@@ -0,0 +1,142 @@
+`, "\n")
+
+func (o *clusterClaimOptions) validateAndComplete() error {
+	if o.namespace == "" {
+		return fmt.Errorf("the Namespace cannot be empty")
+	}
+	if o.clusterSetID == "" {
+		return fmt.Errorf("the ClusterSet ID cannot be empty")
+	}
+	if o.clusterID == "" {
+		return fmt.Errorf("the cluster ID cannot be empty")
+	}
+
+	return nil
+}
+
+func NewClusterClaimCmd() *cobra.Command {
+	command := &cobra.Command{
+		Use:     "clusterclaims",
+		Args:    cobra.MaximumNArgs(0),
+		Short:   "Create two ClusterClaims in a leader or member cluster",
+		Long:    "Create two ClusterClaims in a leader or member cluster. One for ClusterSet and another for the leader or member cluster",
+		Example: clusterClaimExamples,
+		RunE:    clusterClaimRunE,
+	}
+
+	o := &clusterClaimOptions{}
+	clusterClaimOpt = o
+	command.Flags().StringVarP(&o.namespace, "namespace", "n", "", "Namespace of the ClusterClaim")
+	command.Flags().StringVarP(&o.clusterSetID, "clusterset-id", "", "", "ClusterSet ID of the ClusterClaim for the ClusterSet")
+	command.Flags().StringVarP(&o.clusterID, "cluster-id", "", "", "cluster ID of the ClusterClaim for the leader or member cluster")
+
+	return command
+}
+
+func clusterClaimRunE(cmd *cobra.Command, args []string) error {
+	if err := clusterClaimOpt.validateAndComplete(); err != nil {
+		return err
+	}
+
+	kubeconfig, err := raw.ResolveKubeconfig(cmd)
+	if err != nil {
+		return err
+	}
+	restconfigTmpl := rest.CopyConfig(kubeconfig)
+	raw.SetupKubeconfig(restconfigTmpl)
+
+	k8sClient, err := client.New(kubeconfig, client.Options{Scheme: multiclusterscheme.Scheme})
+	if err != nil {
+		return err
+	}
+
+	clusterClaim := &multiclusterv1alpha1.ClusterClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      multiclusterv1alpha1.WellKnownClusterClaimID,
+			Namespace: clusterClaimOpt.namespace,
+		},
+		Value: clusterClaimOpt.clusterID,
+		Name:  multiclusterv1alpha1.WellKnownClusterClaimID,
+	}
+
+	var createErr error
+	if createErr = k8sClient.Create(context.TODO(), clusterClaim); createErr != nil {
+		if errors.IsAlreadyExists(createErr) {
+			createErr = nil
+		} else {
+			return createErr
+		}
+	}
+	defer func() {
+		if createErr != nil {
+			if err := k8sClient.Delete(context.TODO(), clusterClaim); err != nil {
+				panic(err)
+			}
+		}
+	}()
+
+	clustersetClaim := &multiclusterv1alpha1.ClusterClaim{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      strings.ToLower(multiclusterv1alpha1.WellKnownClusterClaimClusterSet),
+			Namespace: clusterClaimOpt.namespace,
+		},
+		Value: clusterClaimOpt.clusterSetID,
+		Name:  multiclusterv1alpha1.WellKnownClusterClaimClusterSet,
+	}
+
+	if createErr = k8sClient.Create(context.TODO(), clustersetClaim); createErr != nil {
+		if errors.IsAlreadyExists(createErr) {
+			createErr = nil
+		} else {
+			return createErr
+		}
+	}
+
+	fmt.Fprintf(cmd.OutOrStdout(), "ClusterClaim \"%s\" created\n", clusterClaimOpt.clusterID)
```

Comment:
We should not print these messages if IsAlreadyExists.

Reply:
Done. Add a new conditional statement for this.